src/config.cpp

```diff
@@ -2273,17 +2273,43 @@ static int updateMaxclients(long long val, long long prev, const char **err) {
             }
             return 0;
         }
+        /* Change the SetSize for the current thread first. If any error, return the error message to the client, 
+         * otherwise, continue to do the same for other threads */
+        if ((unsigned int) aeGetSetSize(aeGetCurrentEventLoop()) <
+                g_pserver->maxclients + CONFIG_FDSET_INCR)
+        {
+            if (aeResizeSetSize(aeGetCurrentEventLoop(),
+                g_pserver->maxclients + CONFIG_FDSET_INCR) == AE_ERR)
+            {
+                *err = "The event loop API used by Redis is not able to handle the specified number of clients";
+                return 0;
+            }
+	    serverLog(LL_DEBUG,"Successfully changed the setsize for current thread %d", ielFromEventLoop(aeGetCurrentEventLoop()));
+        }
+
         for (int iel = 0; iel < cserver.cthreads; ++iel)
         {
+	    if (g_pserver->rgthreadvar[iel].el == aeGetCurrentEventLoop())
+            {
```

Comment:
We're mixing brace styles in this change still.

Lines you touch should be converted to:
if (x > y) {
   ...
}

Note I'm only putting one comment in but there are multiple lines with this issue.

Reply:
Fixed.